CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.0.0-BETA27
```

Comment:
If we want to release this after merging (without another PR to bump the version), we should also update `LIBRARY_VERSION` in `gradle.properties`. `docs/Release.md` describes the steps necessary for a release afterwards.

Reply:
👍  There is another open issue for SyncStatus updates not reflecting the downloading and uploading updates. I'd like to batch those fixes in a single release.